aea/cli/list.py

```diff
@@ -60,20 +83,20 @@ def _get_item_details(ctx, item_type) -> List[Dict]:
 def connections(ctx: Context):
     """List all the installed connections."""
     result = _get_item_details(ctx, "connection")
-    print(format_items(sorted(result, key=lambda k: k['name'])))
+    print(format_items(sorted(result, key=lambda k: k["name"])))
```

Comment:
Cannot use logger here over print ?

Reply:
We decided to use print because this is a large output and logger prepends too much